rosco-web/config/rosco.yml

```diff
@@ -5,7 +5,9 @@ rosco:
   configDir: /some/path/to/config/packer
   jobs:
     local:
-      timeoutMinutes: 20
+      timeoutMinutes: 30
```

Comment:
In Azure, my Windows bakes were fairly consistently timing out at the 20 minute mark. The Windows bakes usually took 20-25 minutes with 2 Chocolatey packages (Chrome and Java 8). I bumped up to 30 minutes to allow for enabling Windows features since these can take time to do. "orphanedJobTimeoutMinutes" was increased by 5 minutes to account for the extra bake time. Having "timeoutMinutes" and "orphanedJobTimeoutMinutes" set to 30 minutes would end up as a bit of a race condition.